components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -166,6 +166,11 @@
     public static final String OPENID_CONNECT = "OpenIDConnect";
     public static final String ENABLE_OPENID_CONNECT_AUDIENCES = "EnableAudiences";
     public static final String OPENID_CONNECT_AUDIENCE = "audience";
+    /*
+     * Maintain a separate parameter "OPENID_CONNECT_IDENTITY_CONFIG_AUDIENCE" for get the audience form the identity.xml
```

Comment:
Lets use Maintain a separate parameter "OPENID_CONNECT_IDENTITY_CONFIG_AUDIENCE" **to** get